src/NUnitConsole/nunit-console/Utilities/TeamCityEventHandler.cs

```diff
@@ -21,7 +21,6 @@
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 // ***********************************************************************
 
-using System;
```

Comment:
I can go either way on this. Often, I exempt System from the requirement that all using statements be needed.